internal/integrationtest/arduino-cli.go

```diff
@@ -126,6 +126,18 @@ func (cli *ArduinoCLI) SketchbookDir() *paths.Path {
 	return cli.sketchbookDir
 }
 
+// CopySketch copies a sketch inside the testing environment and returns its path
+func (cli *ArduinoCLI) CopySketch(t *testing.T, sketchName string) *paths.Path {
```

Comment:
```suggestion
func (cli *ArduinoCLI) CopySketch(sketchName string) *paths.Path {
```

The `cli` object already has `cli.t` you can use it with a `require`-like syntax:

```go
cli.t.NoError(err)
cli.t.Equals(expected, curr)
....
```